expression/integration_test.go

```diff
@@ -1408,7 +1408,7 @@ func (s *testIntegrationSuite2) TestTimeBuiltin(c *C) {
 
 	// for year
 	result = tk.MustQuery(`select year("2013-01-09"), year("2013-00-09"), year("000-01-09"), year("1-01-09"), year("20131-01-09"), year(null);`)
-	result.Check(testkit.Rows("2013 2013 0 1 <nil> <nil>"))
+	result.Check(testkit.Rows("2013 2013 0 2001 <nil> <nil>"))
```

Comment:
MySQL report `1` here, I doubt it might be MySQL's bug, you can propose it on https://bugs.mysql.com/, until they verified this, we should not modify the tests.

Reply:
As for this test, we have found bugs in mysql-server str_to_datetime function and reported to MySQL. https://bugs.mysql.com/bug.php?id=104350